mash_client/cli/job/azure.py

```diff
@@ -0,0 +1,65 @@
+# -*- coding: utf-8 -*-
+
+"""mash client CLI Azure job endpoints using click library."""
```

Comment:
These are not "endpoints" and endpoint has a very clear and only twice overloaded definition.

"""
Simply put, an endpoint is one end of a communication channel. When an API interacts with another system, the touchpoints of this communication are considered endpoints. For APIs, an endpoint can include a URL of a server or service. ... The place that APIs send requests and where the resource lives, is called an endpoint.
"""

Reply:
Fair, to me this minor concern. If it's confusing enough we can create a bug to cleanup the references.